x/qgb/orchestrator/config.go

```diff
@@ -28,10 +28,10 @@ func init() {
 
```

Comment:
@evan-forbes  Parsing the flags in this file is a bit repetitive. However, I couldn't find a way to better structure them.
Please let me know if you have an idea on how to do it better.